components/Steps/EligibilityCriteria.jsx

```diff
@@ -13,16 +14,12 @@ const Step1 = (props) => {
   const [showError, setShowError] = useState(false);
   const onSubmit = (data) => {
     let eligible = true;
+    const sectionQuestions = inputLabels.eligibilityCriteria;
 
     Object.entries(data.eligibilityCriteria).forEach(([key, value]) => {
-      if (key === 'servedLegalNotices' && value === 'Yes') {
-        eligible = false;
-      } else if (key === 'eligibleForLrsgGrants' && value === 'Yes') {
-        eligible = false;
-      } else if (
-        key !== 'servedLegalNotices' &&
-        key !== 'eligibleForLrsgGrants' &&
-        value === 'No'
+      if (
+        value != sectionQuestions[key].validAnswer ||
+        !sectionQuestions[key].validAnswer
```

Comment:
I'm not sure it matters but it might be better to put this part of the conditional first? so:

```ts
!sectionQuestions[key].validAnswer ||
value !== sectionQuestions[key].validAnswer
```